### PR TITLE
fix(connect-popup): remove initTransport from core import

### PR DESCRIPTION
**Pull request by martykan**
## Description

In https://github.com/trezor/trezor-suite/commit/00deffaa63db5cf0143b7760d5bc23b45d30e0f3 `initTransport` was removed. 
However in connect-popup remained a condition to check if it's defined, so initiating core would fail. 

I moved the condition around the old commented-out block for initTransport, so it won't be called for the new version but can be called if it's present for backwards compatibility.

### CODE REVIEW

File: packages/connect-popup/src/index.tsx [hunk 1]
```diff
@@ -418,13 +416,13 @@ const initCoreInPopup = async (
...
-    log.debug('initiating transport with settings: ', payload.settings);
-    reactEventBus.dispatch({ type: 'loading', message: 'initiating transport' });
-    await initTransport(payload.settings);
-    if (disposed) return;
-    */
+    // init transport - deprecated, here for backward compatibility
+    if (initTransport) {
+        log.debug('initiating transport with settings: ', payload.settings);
+        reactEventBus.dispatch({ type: 'loading', message: 'initiating transport' });
+        await initTransport(payload.settings);
```
Comment (mroz22) [inline packages/connect-popup/src/index.tsx:423]: is there really some backwards compatibility issue? this all happens on trezor.io domain

Comment (marekrjpolak) [reply packages/connect-popup/src/index.tsx:423]: Because of custom `connectSrc`?

Comment (martykan) [reply packages/connect-popup/src/index.tsx:423]: Most likely not, but there can always be some edge case, so I thought we might keep it around for a bit until we remove it fully. For example older script in cache

Comment (mroz22) [reply packages/connect-popup/src/index.tsx:423]: ok, `connectSrc` or older cache could be it probably

**Review by mroz22 — APPROVED** [review]
@martykan would you please create and issue and assign it to yourself? to check it after some time and remove it.

**Pull request merged**